packages/x-charts/src/hooks/useTicks.ts

```diff
@@ -236,6 +259,7 @@ export function useTicks(
     tickInterval,
     tickPlacement = 'extremities',
     tickLabelPlacement,
+    tickSpacing,
```

Comment:
You need to update the ChartsXAxis and ChartsYAxis to give them access to this prop